ajax/libs/inferno-component/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "inferno-component",
+  "version": "3.6.4",
+  "filename": "inferno-component.min.js",
+  "license": "MIT",
+  "description": "Provides ES2015 stateful components (with lifecycle events) to Inferno",
+  "author": {
+    "name": "Dominic Gannaway",
+    "email": "dg@domgan.com"
+  },
+  "homepage": "https://infernojs.org/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/infernojs/inferno.git"
+  },
+  "keywords": [
+    "babel",
+    "react",
+    "inferno",
+    "framework",
+    "interfaces",
+    "user interfaces",
+    "html",
+    "renderToString",
+    "server",
+    "dom",
+    "browser",
+    "rollup"
+  ],
+  "npmName": "inferno-component",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/inferno-component!(*.node|*.es2015)*.+(js|css|map)"
```

Comment:
I can't understand this rule, can you tell me what files do you want to fetch and what files you don't want to?
which version is `*.es2015` for?